src/cascadia/TerminalSettingsModel/SshHostGenerator.cpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+
+#include "SshHostGenerator.h"
+#include "../../inc/DefaultSettings.h"
+
+#include "DynamicProfileUtils.h"
+
+static constexpr std::wstring_view SshHostGeneratorNamespace{ L"Windows.Terminal.SSH" };
+
+static constexpr const wchar_t* PROFILE_TITLE_PREFIX = L"SSH - ";

```

Comment:
Could you use `std::wstring_view` for all of these? It makes concatenations more pleasant and performant.